roaring/roaring.go

```diff
@@ -1792,6 +1773,44 @@ func (c *Container) optimize() {
 	}
 }
 
+func (c *Container) unionInPlace(other *Container) {
```

Comment:
let's add a comment that `c.n` may not necessarily be correct after this operation and should be repaired.